docker/Dockerfile

```diff
@@ -19,23 +19,6 @@ FROM ${BASE} AS scilla
 # run a copy -L to unfold the symlinkes, and strip all exes
 RUN mkdir -p /scilla/0/bin2/ && cp -L /scilla/0/bin/* /scilla/0/bin2/ && strip /scilla/0/bin2/*
 
-FROM ubuntu:18.04 as rust_builder
-RUN apt-get update && apt-get install -y protobuf-compiler && \
-apt-get install -y git && \
-apt-get install -y curl && \
-apt-get install -y build-essential && \
-apt-get install -y pkg-config && \
-apt-get install -y libssl-dev && \
-curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs -o install_script.sh && \
-sh install_script.sh -y 
-ENV PATH="/root/.cargo/bin:${PATH}"
-RUN git clone https://github.com/Zilliqa/evm-ds /evm-ds && \
-    cd /evm-ds && \
-    git checkout 70dfb1488e39b7c1e5fab6f040916c32ee714838 && \
-    cargo build --release --package evm-ds && \
-    cargo test --verbose --release --package evm-ds
```

Comment:
did you remove `cargo test` intentionally?

Reply:
Yes, it should run on CI eventually, but not essential for building an image.